components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -397,6 +408,21 @@ class PostList extends React.PureComponent {
         this.updateFloatingTimestamp(visibleStartIndex);
     }
 
+    getNewMessageIndex = () => {
+        return this.state.postListIds.findIndex(
+            (item) => item.indexOf(PostListRowListIds.START_OF_NEW_MESSAGES) === 0
+        );
+    }
+
+    countNewMessages = () => {
+        const mark = this.getNewMessageIndex();
+        if (mark <= 0) {
+            return 0;
+        }
+        const newMessages = this.state.postListIds.slice(0, mark);
+        return newMessages.filter((id) => !(id.startsWith('date-') || id.startsWith('user-') || id.startsWith('CHANNEL'))).length;
```

Comment:
Can you replace this with the utils `isIdNotPost` in `utils/post_utils.jsx` which is ideal for this and it uses pre-defined constants.